menu/menu.lua

```diff
@@ -38,45 +38,51 @@ menu.key_map = {
         state.debugOn = not state.debugOn
     end,
     p = function()
-        state.paused = not state.paused
-        menu.unload()
-        love.audio.play(sounds.chirp_down)
+        menu.load_map()
+
     end,
     up = function()
         cycleMaps.up()
     end,
     down = function()
         cycleMaps.down()
     end,
+    enter = function()
+        menu.load_map()
+
+    end,
     backspace = function()
-        -- If selected map is the same, just unpause...
-        if state.activeMap == mapSelect then
-            state.paused = not state.paused
-            menu.unload()
-            love.audio.play(sounds.chirp_down)
-        -- ...or else load a new map
-        else
-            --Entity.list = {}
-            loadMap(mapSelect)
-
-            state.paused = not state.paused
-            menu.unload()
-            love.audio.play(sounds.chirp_down)
-        end
+        menu.load_map()
```

Comment:
Since we have 3 ways to load a map now, we could probably get rid of the backspace altogether since it's very un-intuitive.

Reply:
Fixed